themes/classic/_dev/css/components/search-widget.scss

```diff
@@ -34,20 +34,82 @@
   }
 }
 
+#close-search,
+.search-cross {
+  display: none;
+}
+#close-search:checked ~ ul.ui-autocomplete.ui-front.ui-widget-content {
+  display: none !important;
+}
+
+ul{
+  &.ui-autocomplete.ui-front {
+    &.ui-widget-content {
+      border: none;
+      border-radius: 0;
+      .ui-menu-item {
+        &:hover {
+          border: none;
+          border-radius: 0;
+        }
+        a {
+          padding-top: .47rem;
+          padding-left: .813rem;
+          padding-bottom: .47rem;
+          &:hover {
+            border: none;
+            border-radius: 0;
+            background: $gray-light-third !important;
+          }
+          &.ui-state-focus{
+            margin: 0 !important;
+            border: 0 !important;
+            background: $btn-warning-color;
+          }
+          .product {
+            font-size: .9rem;
+            font-family: Noto sans,arial,sans-serif;
+            vertical-align: top;
+          }
+        }
+      }
+    }
+  }
+}
 /*** Responsive part ***/
 @include media-breakpoint-down(sm) {
   .header-top {
     .search-widget {
       float: none;
+      background: $gray-light;
+      margin-bottom: .625rem;
       form {
         margin: 0 auto;
         input[type="text"] {
           min-width: inherit;
-          background: white;
+          &:not(:placeholder-shown) ~ .search-cross {
+            display:block;
+          }
         }
       }
     }
   }
+  .search-cross {
+    position: absolute;
+    background: none;
+    border: none;
+    bottom: .0125rem;
+    right: .625rem;
+    color: $gray-searchbar;
+  }
+  ul {
+    &.ui-autocomplete.ui-front {
+      &.ui-widget-content {
+        left: 0 !important;
```

Comment:
same here, why using important and not a better selector?

Reply:
There's no other selector. 
It's located at the end of body